src/game-of-life/setup.md

```diff
@@ -12,17 +12,26 @@ You will need the standard Rust toolchain, including `rustup`, `rustc`, and
 
 [rust-install]: https://www.rust-lang.org/en-US/install.html
 
-## The `wasm32-unknown-unknown` Target
+## `wasm-pack`
 
-Once you have the Rust toolchain installed, you'll want to be able to compile
-Rust programs to WebAssembly, rather than your machine's native code. You can
-enable this by adding the `wasm32-unknown-unknown` target with the following
-command:
+[`wasm-pack` is your one-stop shop for building, publishing, and working with
+Rust-generated WebAssembly that inter-operates with JavaScript.][wasm-pack]
+
+Install `wasm-pack` with this command:
+
+```
+cargo install wasm-pack
+```
+
+## `cargo-generate`
+
+[`cargo-generate` helps you get up and running quickly with a new Rust project
+by leveraging a pre-existing git repository as a template.][cargo-generate]
+
+Install `cargo-generate` with this command:
 
 ```
-rustup update
-rustup install nightly
-rustup target add wasm32-unknown-unknown --toolchain nightly
+cargo install cargo-generate
```

Comment:
This review is out of order because of the way GitHub orders things. So yeah this answers my question

Reply:
Great, my replies are out of order too :)